dateparser_cli/cli.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+import logging
+
+from .fasttext_manager import fasttext_downloader
+from .utils import clear_cache
+
+
+def entrance():
+    dateparser_argparse = argparse.ArgumentParser(
+        description='dateparser download manager.'"
+    )
+    dateparser_argparse.add_argument(
+        '--fasttext',
+        type=str,
+        help='To download a fasttext language detection models. Supported models are "small" and "large"'
+    )
+    dateparser_argparse.add_argument(
+        '--clear',
+        '--clear-cache',
+        help='To clear all cached models',
+        action='store_true'
+    )
+
+    args = dateparser_argparse.parse_args()
+
+    if args.fasttext:
+        fasttext_downloader(args.fasttext)
+    if args.clear:
+        clear_cache()
+        dateparser_argparse.error("dateparser-download: All cache deleted")
```

Comment:
@gavishpoddar I think there are a few issues here now:

- I think it's best to first clear the cache, and then download - otherwise we'll clear what we just downloaded
- Previously you had an else branch with  ``"dateparser-download: You need to specify the command (i.e.: --fasttext or --clear)"`` - I think it would be great to bring it back, using ``dateparser_argparse.error`` to show it.
- For ``dateparser-download: All cache deleted`` message, I think it's best to use ``logging.info`` as before - this is not an error. ``dateparser_argparse.error`` would show a message and terminate the program with a non-succesfull return code, which is not we want here

Reply:
Hey I have @lopuhin, made the changes. Please check and suggest if its fine and what more needs to be done.

Reply:
looks great, thanks @gavishpoddar :+1: